code/datums/limb.dm

```diff
@@ -37,7 +37,8 @@
 	proc/attack_hand(atom/target, var/mob/user, var/reach, params, location, control)
 		if(!target) // fix runtime Cannot execute null.attack hand().
 			return
-
+		if(SEND_SIGNAL(target, COMSIG_ATTACKHAND, user))
+			return
```

Comment:
There is no way in hell that this catches all `atom/proc/attack_hand` calls 😭 
Will probably need a similar garbage a la `Attackby()` to make sure the signal is always sent whenever `atom/proc/attack_hand()` is called...